backend/dataall/modules/worksheets/__init__.py

```diff
@@ -16,8 +18,12 @@ class WorksheetApiModuleInterface(ModuleInterface):
     def is_supported(modes):
         return ImportMode.API in modes
 
+    @staticmethod
```

Comment:
Why do worksheets depend on Feed?

Reply:
Honestly, I have scrupulously just modularize it from main as we could post a message for a worksheet. 

But I have just checked it in the frontend and there is no button to post a message for a worksheet (possible, this feature had gone away even before `WorksheetShare` ). Thought, I might have overlooked it
Do you want me to delete this ?

Reply:
I would remove it for the moment, just to make it clear that it is unused in worksheets